erdpy/cli_validators.py

```diff
@@ -40,6 +45,40 @@ def setup_parser(subparsers: Any) -> Any:
     _add_common_arguments(sub)
     sub.set_defaults(func=do_claim)
 
+    sub = cli_shared.add_command_subparser(subparsers, "validator", "unstake-nodes", "It will unStake nodes")
```

Comment:
Description is a bit different from all the other commands (`it will ...`). But should be OK.

Reply:
refactored.